### PR TITLE
Migrate after_config callback to hook

### PR DESCRIPTION
**Pull request by tuanngocnguyen**
Regards issue: https://github.com/catalyst/moodle-local_envbar/issues/219

### CODE REVIEW

File: classes/hook_callbacks.php [hunk 1]
```diff
@@ -38,4 +39,27 @@ public static function before_standard_top_of_body_html_generation(before_standa
...
+     */
+    public static function after_config(\core\hook\after_config $hook): void {
+        global $CFG;
+
+        if (during_initial_install() || isset($CFG->upgraderunning)) {
+            // Do nothing during installation or upgrade.
+            return;
+        }
+
+        // Hack to avoid breaking messaging tests, as this setting defaults on.
```
Comment (dmitriim) [inline classes/hook_callbacks.php]: Can you please provide a bit more details on what issue are you talking about? What setting is on by default?

Comment (dmitriim) [reply classes/hook_callbacks.php]: Ah I see you just copied it from lib.php

Comment (dmitriim) [reply classes/hook_callbacks.php]: Do we need just call local_envbar_after_config() in this hook to avoid the code duplication?

Comment (tuanngocnguyen) [reply classes/hook_callbacks.php]: Hi @dmitriim ,

I have linked to an issue and update the hook.

File: classes/hook_callbacks.php [hunk 2]
```diff
@@ -16,6 +16,8 @@
 
 namespace local_envbar;
 
+require_once(__DIR__ . '/../lib.php');
```
Comment (dmitriim) [inline classes/hook_callbacks.php]: I would recommend changing this to exact path like 

`require_once($CFG->dirroot . '/local/envbar/lib.php');`

Comment (tuanngocnguyen) [reply classes/hook_callbacks.php]: Hi @dmitriim ,

I have updated the patch

**Pull request merged**